app/src/main/java/org/wikipedia/page/PageFragment.kt

```diff
@@ -1274,8 +1275,11 @@ class PageFragment : Fragment(), BackPressedHandler, CommunicationBridge.Communi
             updateProgressBar(false)
         }
 
-        override fun onError() {
-            avPlayer?.stop()
+        override fun onError(code: Int, extra: Int) {
+            if (avPlayer?.isPlaying == true) {
+                avPlayer?.stop()
+            }
+            FeedbackUtil.showMessage(this@PageFragment, getString(R.string.media_playback_error, code.toString()))
```

Comment:
The error message for the article "Nikola Jokic" will show "Error playing media: 1". 

Do you think the error code "1" make sense to users or can we show the error message without the error code?

Reply:
Yeah, I suppose you're right; done!
I was going back and forth about it. And actually, the error code "1" just means "UNKNOWN", so it's not helpful anyway. If a user sees this error, they can send us a screenshot, and we'll see which article they're on, so we can reproduce the code ourselves.